src/shim.js

```diff
@@ -91,23 +91,39 @@ const commands = {
         completeCommand(command);
     },
 
+    loadPageContent: function (command) {
+        let page = objectSpace[command.target];
+        if(!page) {
```

Comment:
IMO it should, because e.g. `render` (aka `invokeMethod('render')`) returns true or false depending on whether it succeeded or not. That's something we learned the hard way too.
This way, if the return value is clearly described in docs, we can keep the API more consistent even if a bit odd.

Exception here would kill the whole phantom process.